### ISSUE TITLE
Missing ManagementInfrastructure (mi.h) header

### ISSUE THREAD

**Issue by wmgries** [microsoft/win32metadata#434]
Hi folks, 

I already opened a PR a few days ago for this topic, but I noticed that issues are being created for missing APIs, so here's my issue. The MI APIs are used for WMIv2. My reference here for their importance is that PowerShell uses these APIs, rather than older WMI APIs. This blog post [here](https://devblogs.microsoft.com/scripting/should-i-use-cim-or-wmi-with-windows-powershell/) provides some explanation of why.

I've already been using these APIs in a private build of the win32metadata and the [windows-rs](https://github.com/microsoft/windows-rs) language projection. I don't really care about my individual PR if I've done something wrong here, but it'd be nice not to have to do private builds anymore :)

Here's the PR: https://github.com/microsoft/win32metadata/pull/426

Thanks!

**Comment by kennykerr** [comment]
Thanks for the contribution! Hopefully @sotteson1 will have a moment to review the PR soon.

**Comment by mikebattista** [comment]
Thanks. We are also refactoring all the namespaces and looking at what other headers might be missing at the same time.

We'll make sure this is addressed as part of that.

**Comment by wmgries** [comment]
@mikebattista I do have another couple headers that I'm using that I have created a PR for yet. Is it useful for me to create that PR?

**Comment by mikebattista** [comment]
Yes that would be helpful. We basically have to go through the missing headers, identify what functional areas they belong to, then assign them to appropriate namespaces.

If you've already done that for some, that would save us some effort. And whether we accept the PRs as is or not, they will help us prioritize the headers we include first. We are refactoring all the namespaces in parallel so just pick a reasonable starting point and we'll put it in an appropriate place.

If you want to see the latest state of namespaces, build the winmd from the [namespaces ](https://github.com/microsoft/win32metadata/tree/namespaces) branch.

**Comment by sotteson1** [comment]
I committed the PR.

**Comment by mikebattista** [comment]
If this is WMIv2, would it make sense to put them in Windows.Win32.System.Wmi with the other WMI APIs?

I'm looking to move them from Windows.Win32.ManagementInfrastructure down into the new structure for namespaces. These should be with the Wmi namespace or close to it.
